ledfx/devices/rpi_ws281x.py

```
@@ -115,9 +121,27 @@ def deactivate(self):
-                (round(rgb[0]) << 16) | (round(rgb[1]) << 8) | round(rgb[2]),
+        # Apply color mapping function
+        data = self.output_mode.apply(data)
+
+        # Switch between 24bit (RGB) and 32bit (RGBW) output
+        if self.white_mode == "None":
+            # RGB: R, G, B
+            color_func = (
+                lambda c: (round(c[0]) << 16)
+                | (round(c[1]) << 8)
+                | round(c[2])
+            )
```

Ruff (0.15.5)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `color_func` as a `def`

(E731)